bash/gast_to_code/bash_gast_to_code_converter.py

```diff
@@ -10,4 +10,10 @@ def handle_bool(gast):
             return "true"
         else:
             return "false"
+    
+    def handle_log_statement(gast):
+        return "echo"
```

Comment:
Is this intentionally not printing the arguments?

Reply:
Yes, this is how we do it in every language. Arguments are handled by another recursive call to the router